### DISCUSSION TITLE
Compare with a few commits back, not just the last commit

### DISCUSSION THREAD

**Discussion by leotrs** [akaihola/darker#320] [Q&A] [answered]
This is a great project! Is it possible to apply darker to the lines modified since a few commits back, instead of just comparing to the last commit?

**Comment by leotrs** [comment, marked as answer]
Woops, just found the `-r` flag. This is awesome!

**Comment by akaihola** [comment]
Thanks @leotrs! The `-r`/`--revision` flag is quite a new thing – please come back with feedback if you find any flaws in how it works. Also, if it isn't well explained in the documentation, I'd consider that a documentation bug worth fixing.
